usr/share/rear/layout/save/default/600_snapshot_files.sh

```diff
@@ -1,9 +1,54 @@
 # Save a hash of files that would warrant a new rescue image when changed.
+
+# FILES_TO_PATCH_PATTERNS is a space-separated list of shell glob patterns.
+# Files that match are eligible for a final migration of UUIDs and other
+# identifiers after recovery (if the layout recreation process has led
+# to a change of an UUID or a device name and a corresponding change needs
+# to be performed on restored configuration files ).
+# See finalize/GNU/Linux/280_migrate_uuid_tags.sh
+# We should add all such files to CHECK_CONFIG_FILES - if they change,
+# we risk inconsistencies between the restored files and recreated layout,
+# or failures of UUID migration.
+
+local file obj final_file symlink_target
+
+# The patterns are relative to /, change directory there
+# so that the shell finds the files during pathname expansion
+pushd / >/dev/null
+# The variable expansion is deliberately not quoted in order to perform
+# pathname expansion on the variable value.
+for file in $FILES_TO_PATCH_PATTERNS ; do
+    final_file="/$file"
+    IsInArray "$final_file" "${CHECK_CONFIG_FILES[@]}" && continue
+    # Symlink handling: replace symlinks by targets (not strictly necessary, but consistent
```

Comment:
I need to delete "replace symlinks by targets (not strictly necessary, but consistent with 280_migrate_uuid_tags.sh)", because I deleted the corresponding code.